e2e/constants.ts

```diff
@@ -0,0 +1,96 @@
+import invariant from '../src/utils/invariant.js';
+
+export type Datasource = { id: string };
+export type Workbook = { id: string; defaultViewId: string };
+
+export type PulseDefinition = { id: string; metrics: Array<PulseMetric> };
+export type PulseMetric = { id: string };
+
+type EnvironmentData = {
+  servers: {
+    [url: string]: {
+      sites: {
+        [name: string]: {
+          datasources: {
+            [name: string]: Datasource;
+          };
+          workbooks: {
+            [name: string]: Workbook;
+          };
+          pulse: {
+            definitions: {
+              [name: string]: PulseDefinition;
+            };
+          };
+        };
+      };
+    };
+  };
+};
+
+const environmentData: EnvironmentData = {
+  servers: {
+    'https://10ax.online.tableau.com': {
+      sites: {
+        'mcp-test': {
+          datasources: {
+            'Superstore Datasource': { id: '2d935df8-fe7e-4fd8-bb14-35eb4ba31d45' },
+          },
+          workbooks: {
+            Superstore: {
+              id: '222ea993-9391-4910-a167-56b3d19b4e3b',
+              defaultViewId: '9460abfe-a6b2-49d1-b998-39e1ebcc55ce',
+            },
+          },
+          pulse: {
+            definitions: {
+              'Tableau MCP': {
+                id: '9ad098f4-49cf-4e8a-bec0-0ca803091dd0',
+                metrics: [{ id: 'fd6c4aa0-f6d3-469e-b75b-d597435ae199' }],
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+};
```

Comment:
This object here contains info about the data source, workbook, and Pulse metric that exist on this site. It gets used to provide the correct tool arguments and for some test assertions. If we wanted to run the tests against other sites, we'd need to update this object.